Pipfile

```diff
@@ -32,4 +32,4 @@ nplusone = "*"
 codecov = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.8"
```

Comment:
Could you also change the Dockerfile to point to `python:3.8.6`?

I think we should also update the docker-compose file to use the `postgres:13.1` image instead (you'll have to delete the container and volumes associated because the data format from 9.6 is not compatible).

Also, if the "saas" app in the docker-compose file could use `node:14.15.1`, that would be great.

I tested with PR would these versions and it seemed to work fine.

docker-compose.yml (notice there's a new password env var for the DB and the URL is changed as well):

```
app:
  build: .
  volumes:
    - ./tock:/tock
  links:
    - db
  working_dir: /tock
  entrypoint: python /tock/docker_entrypoint.py
  environment:
    - PYTHONUNBUFFERED=yup
    - DATABASE_URL=postgres://tock_user:tock_password@db/tock
    - RUNNING_IN_DOCKER=yup
    - DJANGO_SETTINGS_MODULE=tock.settings.dev
    # - NEW_RELIC_LICENSE_KEY=get-from-vcap-user-provided-service (not to be used for local development)
  stdin_open: true
  tty: true
  ports:
    - "8000:8000"
  command: "python manage.py runserver 0.0.0.0:8000"
db:
  image: postgres:13.1
  environment:
    - POSTGRES_DB=tock
    - POSTGRES_USER=tock_user
    - POSTGRES_PASSWORD=tock_password
sass:
  image: node:14.15.1
  volumes:
    - .:/tock
    - /tock/node_modules
  working_dir: /tock
  entrypoint: npm start
```

Reply:
Thanks @gtirloni, I've made these updates in 5323754.